docs/core/deploying/trim-self-contained.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Trim self-contained deployments and executables
+description: Learn how to trim self-contained deployments and executables to reduce their size. 
+author: jamshedd
+ms.author: jamshedd
+ms.date: 01/23/2020
+---
+# Trim self-contained deployments and executables
+
+When publishing self-contained deployments (SCD) or self-contained executables (SCE), the .NET Core runtime is bundled together with the application. This bundling adds a significant amount of content to your packaged application.
+
+When it comes to deploying your application, size is often an important factor. Keeping the size of the package application as small as possible is typically a goal for application developers.
+
+Depending on the complexity of the application, only a subset of the runtime is required to run the application. These unused parts of the runtime are unnecessary and can be trimmed from the packaged application.
+
+> [!NOTE]
+> Trimming is an experimental feature in .NET Core 3.0. This feature isn't available for framework-dependent deployments (FDDs) and framework-dependent executables (FDEs) since they don't include .NET Core assemblies.
+
+## Trim your application
+
+The following example shows how to trim your application using the [dotnet publish](../tools/dotnet-publish.md) command:
```

Comment:
Consider adding instructions on doing the same within Visual Studio. (Perhaps in another PR)

Reply:
@jamshedd is this also supported in VS?

Reply:
Yes, this is supported in VS from 16.4 onward. Here's a screenshot from 16.5 Preview 4.

<img width="567" alt="Screen Shot 2020-02-26 at 1 15 34 PM" src="https://user-images.githubusercontent.com/2212879/75388616-26f4b280-589a-11ea-942b-daa937368c35.png">

Reply:
and what about VS for Mac?